Documentation/commands.md

```diff
@@ -127,14 +134,58 @@ sha512-fa1cb92dc276b0f9bedf87981e61ecde
 
 ## Running Containers
 
-Work in progress. Please contribute!
+Rocket can run ACIs based on name, hash, local file on disk or URL. If an ACI hasn't been cached on disk, Rocket will attempt to find and download it.
+
+By default, signatures are checked before an ACI will be run &mdash; be sure to trust the appropriate key before running.
```

Comment:
Right now they're checked when an ACI is _fetched_, not necessarily run (i.e. if it's already local)